docs/apis/plugintypes/format/_examples/lib.php

```diff
@@ -0,0 +1,106 @@
+
+defined('MOODLE_INTERNAL') || die();
+require_once($CFG->dirroot . '/course/format/lib.php');
```

Comment:
What's in `/course/format/lib.php` that we need!?

Reply:
The course_get_format method is there, but it is not necessary anymore as it is calling a static method. I remove it right now. Thanks!